tests/mock_crust/churn.rs

```diff
@@ -93,8 +93,27 @@ impl ExpectedGets {
         self.messages.insert((data_id, msg_id, src, dst));
     }
 
+    /// Sends a request from the client, and adds the expectation that the nodes belonging to `dst`
+    /// receive the message.
+    fn client_send_and_expect(&mut self,
```

Comment:
So much of this is identical to `send_and_expect()` that I think the common part should be abstracted to a separate function.